opm/simulators/linalg/bda/MultisegmentWellContribution.hpp

```diff
@@ -51,7 +55,11 @@ class MultisegmentWellContribution
     unsigned int Mb;                         // number of blockrows in C, D and B
 
 #if HAVE_CUDA
+#if USE_HIP
```

Comment:
i think you should introduce a type alias. saves one if nest and make things more clear.